bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/model/UniFiController.java

```diff
@@ -61,18 +61,30 @@
 
     private final int port;
 
+    private String csrfToken;
```

Comment:
Just a small remark from my side, I would not interleave this mutable field between the other immutable fields.